plugin-redfish/rfphandler/telemetry_test.go

```diff
@@ -0,0 +1,56 @@
+//(C) Copyright [2020] Hewlett Packard Enterprise Development LP
+//
+//Licensed under the Apache License, Version 2.0 (the "License"); you may
+//not use this file except in compliance with the License. You may obtain
+//a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+//WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+//License for the specific language governing permissions and limitations
+// under the License.
+// Packahe rfphandler ...
+package rfphandler
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/ODIM-Project/ODIM/plugin-redfish/config"
+	"github.com/ODIM-Project/ODIM/plugin-redfish/rfpresponse"
+	iris "github.com/kataras/iris/v12"
+	"github.com/kataras/iris/v12/httptest"
+	"net/http"
+	"testing"
+)
+
+func mockUpdateTrigger(username, password, url string, w http.ResponseWriter) {
+	w.WriteHeader(http.StatusOK)
+}
+
+func TestUpdateTrigger(t *testing.T) {
+	config.SetUpMockConfig(t)
+
+	deviceHost := "localhost"
+	devicePort := "1234"
+	ts := startTestServer(mockUpdateTrigger)
```

Comment:
what does this call do ?

Reply:
For all UT under plugin, we have a test server running which handles our server calls https://github.com/ODIM-Project/ODIM/blob/development/plugin-redfish/rfphandler/biosSettings_test.go#L155